src/index.js

```diff
@@ -1,54 +1,80 @@
 'use strict';
 
-const $tagsInput = document.getElementById('tags-input');
-const $tagsList = document.getElementById('tags-list');
-const tagsArr = ['1', 'super tag'];
 
-$tagsInput.addEventListener('keyup', ({key, target}) => {
-  if (key === 'Enter' && target.value.trim()) {
-    const $el = createTagElement(target.value);
-    const isElementExist = tagsArr.includes(target.value);
+function  tagsMechanics(i, defaultArr = []) {
```

Comment:
two spaces before a function name
👎 
all function or methods must be "verb"!
👎